lib/DerbyViewPlugin.js

```diff
@@ -26,6 +26,13 @@ DerbyViewsPlugin.prototype.apply = function(compiler) {
         const appPath = require.resolve(requirePath, { paths: [rootPath] });
         if (!process.env.DERBY_HMR) {
           const app = require(appPath);
+          // scriptUrl and scriptMapUrl need to be defined
+          // but don't matter as not used downstream
+          app.scriptUrl = `${app.name}.views.js`;
+          app.scriptMapUrl = `${app.name}.views.map.js`;
+          // writes serialized views to <app>/derby-serialized directory
+          // used to load pre-compiled views for faster server start
+          app.serialize();
```

Comment:
I'd suggest only running this when Webpack's in production mode, since in dev we want the server to pick up latest views on each server restart.